kafka-connect-azure/src/test/java/io/confluent/connect/StorageSinkTestBase.java

```diff
@@ -0,0 +1,97 @@
+package io.confluent.connect;
+
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.connect.sink.SinkTaskContext;
+import org.junit.After;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class StorageSinkTestBase {
```

Comment:
This looks like it was copied from the `core` module. You can instead add that in the `pom.xml` as a dependency to get this class.

Reply:
Good one. Done.